go.sum

```diff
@@ -0,0 +1,103 @@
+github.com/BurntSushi/toml v0.3.0/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
```

Comment:
You may have done this already but just run `go mod tidy`

Reply:
yes - but then it removes the dependency with go-bindata, which is actually needed. I imagine it has something to do with the fact that it go-bindata doesn't use go modules. I checked that doing a go mod tidy, it only removes this dependency (there are no more unused dependencies in go.sum)